### PR TITLE
Add tag atom

### PR DESCRIPTION
**Pull request by jlight99**
This fulfills task 4 of issue #72 

To test yourself:
include the Tag component anywhere and import it
ex. `<Tag text="One size"></Tag>`

Screenshot:
blueprint_tag

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -18,14 +18,11 @@ function App(): JSX.Element {
       <Switch>
         
         <Route path='/signup' strict component={SignUpModal}></Route>
-        <Route path='/confirmation' component={() => <ConfirmationModal email="anna@pregnancycentre.ca"/>}></Route>
```
Comment (jlight99) [inline client/src/App.tsx:21]: the changes to App.tsx removes unused endpoints that were added for testing purposes when building the auth UI

**Review by guy-ro — APPROVED** [review]
LGTM

File: client/src/components/atoms/style/Tag.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.tag {
+    width: -webkit-fit-content;
+    width: -moz-fit-content;
+    width: fit-content;
+    padding: 11px;
+    background-color: #EDEDED;
```
Comment (meganniu) [inline client/src/components/atoms/style/Tag.scss]: could we make `#EDEDED` a global scss variable in `_config.scss`?

**Review by meganniu — APPROVED** [review]
LGTM, thanks!!!

File: client/src/components/atoms/style/Tag.scss (hunk 2 above)
Comment (jlight99) [reply client/src/components/atoms/style/Tag.scss]: done!

**Pull request merged**